src/rmp/src/rmp.tcl

```diff
@@ -52,19 +51,18 @@
 sta::define_cmd_args "restructure" { \
                                       [-slack_threshold slack]\
                                       [-depth_threshold depth]\
-                                      [-liberty_file liberty_file]\
                                       [-target area|timing]\
-                                      [-tielo_pin tielow_pin]\
-                                      [-tiehi_pin tiehigh_pin]
+                                      [-liberty_file liberty_file]\
```

Comment:
Is this taken from sta now?

Reply:
There should not be a -liberty_file argument to the command

Reply:
Had removed this but had to bring this back since ABC takes only one liberty file. Private regressions are failing since we need more than one liberty file there. So we have to provide "DONT_USE_SC_LIB" just as we do in synth.tcl for ABC.

Reply:
that is disgusting

Reply:
@mathursanjiv ABC supports multiple liberty files, there is no limitation. The current version of YOSYS the OpenROAD-flow is using is the one that can only handle one liberty file, but that has since been changed. So, there should be no problem providing ABC with more than one.  One issue I do see in using the original liberty files, is the dont_use flags are not set in those and ABC may use cells that should have been marked dont_use.

Reply:
@arlpetergadfort can you point me to some testcase which runs with multiple liberty files? May be its version or command difference. For me ABC could not find master from one of liberty file provided. After using integrated liberty file all regressions went through without that error.

Reply:
I'm not sure there are any examples at the moment.
Here is the code YOSYS uses to build the abc script: https://github.com/YosysHQ/yosys/blob/862e84eb3d85dfaacc7025b33c7dbe038bfff2d5/passes/techmap/abc.cc#L714
ABC seems to choke on some of the memory liberty files.

Maybe the better approach would be to only select the liberty files that contain non-blocks. Ie. if the liberty file only contains hard macros (or all don't use cells), ABC doesn't need that file. I suspect in most cases you would end up providing just one or two liberty files instead of all of them.

Reply:
I'll say it again: the liberty file should be obvious by looking at the gates that are included in the restructure blob. I do not believe there is any example in all of our tests that have more than one standard cell library.

Reply:
With single lib, gf12/aes failed and few more from private regression.